src/executables/master_server_monitor.cpp

```diff
@@ -218,12 +215,16 @@ server_monitor::check_server_revived (const std::string &server_name)
     {
       if (errno == ESRCH)
 	{
-	  fprintf (stdout, "[SERVER_REVIVE_DEBUG] : Can't find server process. pid : %d\n", entry->second.get_pid());
+	  _er_log_debug (ARG_FILE_LINE,
+			 "[Server Monitor] [%s] Revived server process can not be found. Server monitor will try to revive server again. (pid : %d)",
+			 entry->first.c_str(), entry->second.get_pid());
 	  produce_job_internal (job_type::REVIVE_SERVER, -1, "", "", entry->first);
 	}
       else
 	{
-	  fprintf (stdout, "[SERVER_REVIVE_DEBUG] : Server revive failed. pid : %d\n", entry->second.get_pid());
+	  _er_log_debug (ARG_FILE_LINE,
+			 "[Server Monitor] [%s] Failed to revive server due to unknown error from kill() function. Server process will not be revived. (pid : %d)",
```

Comment:
```suggestion
			 "[Server Monitor] [%s] Failed to revive server due to unknown error from kill() function. It will no longer be revived automatically. (pid : %d)",
```